components/ws-daemon/pkg/content/service.go

```diff
@@ -218,6 +218,10 @@ func (s *WorkspaceService) InitWorkspace(ctx context.Context, req *api.InitWorks
 			GID:     wsinit.GitpodGID,
 		}
 		if req.UserNamespaced {
+			opts.IdMappings = []archive.IDMapping{
+				{ContainerID: 0, HostID: wsinit.GitpodUID, Size: 1},
+				{ContainerID: 1, HostID: 100000, Size: 65534},
+			}
```

Comment:
Could you elaborate on it? Does it mean that:
- [0,1) in container should map to `GitpodUID` on host
- and [1,65534 + 1) in container should map to 100000 (root?) on host

Reply:
| container | host |
|---|---|
| `0` | `33333` |
| `1 - 65534` | `100000 - 165534`